src/app/loadout-drawer/loadout-utils.ts

```diff
@@ -541,24 +562,9 @@ export function isMissingItems(
   loadout: Loadout
 ): boolean {
   for (const loadoutItem of loadout.items) {
-    const info = getResolutionInfo(defs, loadoutItem.hash);
-    if (!info) {
-      // If an item hash is entirely missing from the database, we show that
-      // there is a missing item but can't offer a replacement (or even show
-      // which item went missing), but we can maybe add a migration in `oldToNewItems`?
+    if (!findItemForLoadout(defs, allItems, storeId, loadoutItem)) {
       return true;
     }
-    if (info.instanced) {
-      if (!allItems.some((item) => item.id === loadoutItem.id)) {
-        return true;
-      }
-    } else {
-      // The vault can't really have uninstanced items like subclasses or emblems, so no point
-      // in reporting a missing item in that case.
-      if (storeId !== 'vault' && !getUninstancedLoadoutItem(allItems, info.hash, storeId)) {
-        return true;
-      }
-    }
```

Comment:
This was here to address #8388 -- with this change we get a ton of unactionable warning triangles in the vault emblem dropdown for loadouts with subclasses or emblems.